benchmarks/time_refine_type03.c

```diff
@@ -36,17 +36,17 @@ int max_ref_level = 0;
 
 /* This function refines every element */
 static int
-t8_basic_adapt_refine_type (t8_forest_t forest, t8_locidx_t which_tree, t8_eclass_scheme_t *ts, const int is_family,
-                            int num_elements, t8_element_t *elements[])
+t8_basic_adapt_refine_type (t8_forest_t forest, t8_locidx_t which_tree, t8_eclass_t tree_class,
+                            const t8_scheme_c *scheme, const int is_family, int num_elements, t8_element_t *elements[])
 {
   int level;
   int type;
   int dim;
 
-  T8_ASSERT (!is_family || num_elements == t8_eclass_num_children[ts->eclass]);
+  T8_ASSERT (!is_family || num_elements == t8_eclass_num_children[tree_class]);
 
-  dim = t8_eclass_to_dimension[ts->eclass];
-  level = t8_element_level (ts, elements[0]);
+  dim = t8_eclass_to_dimension[tree_class];
+  level = t8_element_get_level (forest, tree_class, elements[0]);
   if (level >= max_ref_level) {
     return 0;
   }
```

Comment:
```suggestion
  const int dim = t8_eclass_to_dimension[tree_class];
  const int level = t8_element_get_level (forest, tree_class, elements[0]);
  if (level >= max_ref_level) {
    return 0;
  }
```
please update line 54, too.